samples/webextension/glean.js

```diff
@@ -0,0 +1,789 @@
+var Glean =
```

Comment:
Why did you need to copy this here? Not possible to required the glean code?

Reply:
Nope, it's not possible. Apparently files that we include need to live in a children directory of the one with the manifest.